R/here.R

```diff
@@ -1,6 +1,6 @@
 #' @rdname importfunctions
 #' @export
-here <- function(..., .from, .library = .libPaths()[1L],
+here <- function(.from, ..., .library = .libPaths()[1L],
```

Comment:
is this a breaking change? The idea was that it reads funny if you without specifying ".from"...

Reply:
Ok, sounds good.

In theory, a call to `import::from` with the right environment set should also work, but I guess it's not too important. Or at least we can see if it becomes important, and not worry about it now.

Reply:
Agreed, but that would be way harder (and I worry, less robust) to check for. I also think that for the end user, telling them that any inner import statement should be a `here()` statement is simpler messaging than guidelines on which environments should work.

Reply:
✅